ui/raidboss/data/03-hw/dungeon/sohm_al.js

```diff
@@ -72,25 +54,8 @@
     },
     {
       id: 'Sohm Al Tioman Meteor',
-      regex: Regexes.headMarker({ id: '0007' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'place meteor on edge',
-            de: 'Meteor an Kante ablegen',
-            fr: 'Météore à placer sur le côté',
-          };
-        }
-      },
-      tts: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'meteor',
-            de: 'meteor',
-            fr: 'météore',
-          };
-        }
-      },
+      regex: Regexes.headMarker({ id: '0007', capture: false }),
+      response: Responses.meteorOnYou(),
```

Comment:
I feel like this is not as good as what it is replacing.

Reply:
I can understand that you would loose the "on edge" part of the trigger but i think it still better to have it generalized (as all other meteor trigger also dont include a "on X" position part, e.g. t9).

Reply:
I guess all meteors go out, more or less.  Ok, I agree here, then!